packages-ts/gauntlet-solana-contracts/src/lib/scalarMult.ts

```diff
@@ -0,0 +1,172 @@
+// This file contains vendoring of tweetnacl-js' scalarMult
```

Comment:
We should probably think to move it under `/src/vendor` or similar.

This looks like something that could be useful for other Gauntlets :) Maybe consider putting it in the core package.

Reply:
Move all these under `/core`. This should be moved into `gauntlet-core` eventually